Activities/Cryptography/UiPath.Cryptography.Activities/DecryptFile.cs

```diff
@@ -69,6 +70,22 @@ public partial class DecryptFile : CodeActivity
         [LocalizedDescription(nameof(Resources.Activity_DecryptFile_Property_ContinueOnError_Description))]
         public InArgument<bool> ContinueOnError { get; set; }
 
+        [Browsable(false)]
+        [DefaultValue(null)]
+        [LocalizedCategory(nameof(Resources.Input))]
+        [LocalizedDisplayName(nameof(Resources.Activity_DecryptFile_Property_InputFile_Name))]
+        [LocalizedDescription(
+            nameof(Resources.Activity_DecryptFile_Property_InputFile_Description))]
+        public InArgument<IResource> InputFile { get; set; }
+
+        [Browsable(false)]
+        [DefaultValue(null)]
+        [LocalizedCategory(nameof(Resources.Output))]
+        [LocalizedDisplayName(nameof(Resources.Activity_DecryptFile_Property_DecryptedFile_Name))]
+        [LocalizedDescription(nameof(Resources
```

Comment:
Why 2 lines here, resource keys should be on line line even if they are longer.

Also see comments left in the view model.

Reply:
resolved